.github/workflows/cla-assistant.yaml

```diff
@@ -0,0 +1,48 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
```

Comment:
IMO this is _way_ too permissive for a workflow that is triggered on pull_request_target that uses a third-party action. This permission can:
* change what claims are sent to OIDC providers, and what order they're sent in (causes problem in conjunction with AWS wildcard conditions)
* approve workflow runs for external contributors
* trigger workflow dispatches (expected)